internal/plugins/cache.go

```diff
@@ -63,7 +63,7 @@ func saveCache() error {
 	}
 	cacheMoot.Lock()
 	defer cacheMoot.Unlock()
-	if err := os.MkdirAll(filepath.Dir(CachePath), 0744); err != nil {
+	if err := os.MkdirAll(filepath.Dir(CachePath), 0o744); err != nil {
```

Comment:
@fasmat This is the first time I see this one. What does the `0o744` permission means?